packages/design-tokens/src/index.umd.ts

```diff
@@ -0,0 +1,5 @@
+import './light/_index.scss';
+import './dark/_index.scss';
```

Comment:
```suggestion
import './index.scss';
```
That should do the trick?

Reply:
strange but it works.